R-TypeGame/Systems/Move/MoveSystemLeft/MoveSystemLeft.cpp

```diff
@@ -34,6 +34,8 @@ void MoveSystemLeft::updateLeftPosition(ECS::Registry& entityManager, int idxPac
         if (!position || !speed)
             return;
 
+        if (position->x - speed->speedX < 0)

```

Comment:
I think you might set it to 0 instead of returning, because there are worlds where the starship won't be able to get close to a wall

Reply:
Okay but that will be an other hitbox system no ?

Reply:
No, I'm just saying that, for example, if the pos is 40, and your speed 41, you won't be able to get closer to the wall just because the speed is 1 to high, for me, in that case, just set the pos to 0 👍

Reply:
Oh ok I understand.